sticker-utils/tensorflow/write-cnn

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
I think this is also a good excuse to fully move to Python 3. Could you change the shebang to

```
#!/usr/bin/env python3
```

And the `coding` line can be removed, because Python 3's default encoding is UTF-8.

Reply:
The model files also feature `#!/usr/bin/env python`. Is there any use to it?

Reply:
Nope, they are accidental side-effects of copy paste. They can definitely be removed.

Reply:
In his PR?

Reply:
Better do do it in a separate PR.